packages/cdk8s-cli/BUILD.bazel

```diff
@@ -0,0 +1,116 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@build_bazel_rules_nodejs//:index.bzl", "copy_to_bin")
+load("//tools:defaults.bzl", "jest_test", "pkg_npm", "ts_library")
+load("//tools:package-json/package_json.bzl", "package_json")
+
+package_json(
+    name = "package_json",
+    package_name = "cdk8s-cli",
+    bin = {
+        "cdk8s": "bin/cdk8s.js",
+    },
+    desc = "CDK for Kubernetes CLI",
+    experimental = True,
+    peer_deps = {
+        "codemaker": "^0.22.0",
+        "yargs": "^15.1.0",
+    },
+    deps = {
+        "codemaker": "^0.22.0",
+        "sscaff": "^1.2.0",
+        "yargs": "^15.1.0",
+    },
+)
+
+ts_library(
+    name = "cdk8s_cli",
+    srcs = glob(
+        [
+            "lib/**/*.ts",
+            "bin/**/*.ts",
+        ],
+        exclude = ["test/**"],
+    ),
+    deps = [
+        "//packages/cdk8s:lib",
+        "@npm//@aws-cdk/core",
+        "@npm//@types/fs-extra",
+        "@npm//@types/json-schema",
+        "@npm//@types/yargs",
+        "@npm//@types/yaml",
+        "@npm//codemaker",
+        "@npm//sscaff",
+    ],
+)
+
+ts_library(
+    name = "tests",
+    testonly = True,
+    srcs = glob(["test/**/*.ts"]),
+    deps = [
+        ":cdk8s_cli",
+        "@npm//@types/fs-extra",
+        "@npm//@types/json-schema",
+        "@npm//codemaker",
+    ],
+)
+
+filegroup(
+    name = "test_sources",
+    testonly = True,
+    srcs = [":tests"],
+    output_group = "es5_sources",
+)
+
+filegroup(
```

Comment:
again feels like this is duplicated with how all the examples are configured. definitely seems like an opportunity for reuse

Reply:
Agreed, once all functionality is up and running, consolidation is job one 😄